model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/JpaUpdate13_0_0_MigrateDefaultRoles.java

```diff
@@ -48,34 +48,34 @@ protected void generateStatementsImpl() throws CustomChangeException {
                 // create new default role
                 new InsertStatement(null, null, database.correctObjectName("KEYCLOAK_ROLE", Table.class))
                     .addColumnValue("ID", id)
-                    .addColumnValue("CLIENT_REALM_CONSTRAINT", entry.getValue())
+                    .addColumnValue("CLIENT_REALM_CONSTRAINT", entry.getKey())
                     .addColumnValue("CLIENT_ROLE", Boolean.FALSE)
                     .addColumnValue("DESCRIPTION", "${role_" + roleName + "}")
                     .addColumnValue("NAME", roleName)
-                    .addColumnValue("REALM_ID", entry.getValue())
-                    .addColumnValue("REALM", entry.getValue())
+                    .addColumnValue("REALM_ID", entry.getKey())
+                    .addColumnValue("REALM", entry.getKey())
```

Comment:
Why is this change here?

Reply:
The `REALM` column was removed in one of  previous releases in this [commit](https://github.com/keycloak/keycloak/commit/185075d37305463c4e8e82673bb5f2799e094346#diff-b8c40559721894bb6d1c59b1df39894b1f773f903582bf68e5163c0563195127L86-L89). It could then have any value, it doens't affect anything. We might consider dropping the column in the future.

Reply:
https://github.com/keycloak/keycloak/issues/9164